modules/ROOT/pages/apikit-4-prevent-fault-task.adoc

```diff
@@ -1,13 +1,13 @@
-= To Prevent the SOAP Fault
+= Prevent the SOAP Fault
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you resolve the SOAP fault you generated. You add the Transform Message component to the flow and specify a message in DataWeave for responding to the request, effectively implementing the operation. In the DataWeave code, you use the proper namespace to avoid a faultcode at the beginning of the response to the request.
+After resolving the xref:apikit-4-soap-fault-task.adoc[SOAP fault] generated, add the Transform Message component to the flow and specify a message in DataWeave for responding to the request and implementing the operation. In the DataWeave code, use the proper `namespace` to avoid a fault code at the beginning of the response to the request.
 
-The Transform Message component works on the process side after you hit the endpoint defined in the WSDL. This action triggers execution of the main flow of the Mule application, which then calls the backend flow.
+The Transform Message component works on the process side after you hit the endpoint defined in the WSDL. This action triggers an execution of the main flow of the Mule application, which calls the backend flow.
```

Comment:
```suggestion
The Transform Message component works on the process side after you hit the endpoint defined in the WSDL. This action triggers the execution of the main flow of the Mule application, which calls the backend flow.
```

Reply:
Is it necessary to say "works on the process side"? Do "you" hit the endpoint? Is hit the correct verb?

Reply:
Hi, Hanna! 
After adding the transform message in the flow, I think it's relevant to mention its location in the process. 
Please, let me know if you think this information is not relevant.
Regarding "hit", it is too colloquial, I guess. "Reach" seems to be a better option.
Thanks!